widgets/subscriptionListWidget.js

```diff
@@ -51,6 +51,14 @@ class SubscriptionListWidget extends Widget {
 
   handleMessage(topic, msg) {
     var message = msg.toString();
+
+    var subscribedTopic = this.widgetData.topic;
+
+    var regex = new RegExp(subscribedTopic.replace("\\+", "[^/]+").replace("#", ".+"));
```

Comment:
A comment here would make it easier to understand why we are replacing and testing things